packages/babel-preset-udemy-website/tests.js

```diff
@@ -12,7 +12,23 @@ console.log('test babel-preset-udemy-website');
 fs.readdirSync(join(__dirname, 'tests/source')).filter(name => name.endsWith('.js')).forEach(name => {
     const result = babel.transformFileSync(join(__dirname, 'tests/source', name), preset).code;
     const expected = fs.readFileSync(join(__dirname, 'tests/result', name), { encoding: 'UTF-8' });
+
     // eslint-disable-next-line no-console
     console.log(`\tcheck ${name}`);
+
+    // Uncomment this to debug the tests:
+    //
+    // /* eslint-disable no-console */
```

Comment:
You know what would be neat? If you accepted a command line argument to toggle this behavior instead of commenting the code out.

Reply:
Heh, we're recreating Jest's snapshot functionality :-P

Well, your wish is my command :-P

Reply:
Please review the commit named "Added --verbose and --update as options to the tests.js command", @cansin :)